### PR TITLE
inforos.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you please also search for subdomains?

**Comment by galeksandrp** [comment]
Done.

File: src/chrome/content/rules/inforos.ru.xml [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+ns1.inforos.ru ¹
+ns2.inforos.ru ¹
+xhgui.inforos.ru different content
+
+
+¹ mismatch
+-->
 <ruleset name="inforos.ru">
 	<target host="inforos.ru" />
 	<target host="www.inforos.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/inforos.ru.xml]: They have mixed content issues.

Comment (galeksandrp) [inline src/chrome/content/rules/inforos.ru.xml]: Adde `platform="mixedcontent"`

**Pull request merged**